releasenotes/notes/add-serializable-parametric-pulse-31490c4d2cc49ec6.yaml

```diff
@@ -0,0 +1,30 @@
+---
+features:
+  - |
+    New pulse base class :class:`~SymbolicPulse` has been added. This is a
+    drop-in replacement of the conventional :class:`~ParametricPulse` which has been deprecated.
```

Comment:
For someone reading the release notes who has defined a custom `ParametricPulse`, I am not sure "drop-in" would be accurate. They will need to migrate their pulse definition. The built in subclasses like Gaussian are effectively drop-in replacements.

Reply:
Dropped.